php-frontend/src/test/java/org/sonar/php/tree/impl/expression/CastExpressionTreeTest.java

```diff
@@ -39,4 +41,22 @@ public void test() throws Exception {
     assertThat(expressionToString(tree.expression())).isEqualTo("$a");
   }
 
+  @Test
+  public void prefixed_cast_expression() {
+    PrefixedCastExpressionTree tree = parse("b'abc'", PHPLexicalGrammar.CAST_TYPE);
+    assertThat(tree.is(Kind.PREFIXED_CAST_EXPRESSION)).isTrue();
+    assertThat(tree.prefix().text()).isEqualTo("b");
+    assertThat(expressionToString(tree.expression())).isEqualTo("'abc'");
```

Comment:
This looks strange to me. Why not include the prefix?

Reply:
Because it's `expressionToString(tree.expression())` and not `expressionToString(tree)`.
I'll add an assertion about `expressionToString(tree)` , so no one can mix up.